### ISSUE TITLE
qa: Intermittent "Transaction too large" failure in feature_taproot.py

### ISSUE THREAD

**Issue by hebasto** [bitcoin/bitcoin#20330]
https://cirrus-ci.com/task/6235606382542848?command=ci#L289:
```
2020-11-05T23:19:19.911000Z TestFramework (ERROR): JSONRPC error
Traceback (most recent call last):
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/test_framework.py", line 126, in main
    self.run_test()
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/feature_taproot.py", line 1448, in run_test
    self.nodes[1].sendtoaddress(address=addr, amount=int(self.nodes[1].getbalance() * 70000000) / 100000000)
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/coverage.py", line 47, in __call__
    return_val = self.auth_service_proxy_instance.__call__(*args, **kwargs)
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/authproxy.py", line 146, in __call__
    raise JSONRPCException(response['error'], status)
test_framework.authproxy.JSONRPCException: Transaction too large (-6)
```

**Comment by MarcoFalke** [comment]
#20292

**Comment by hebasto** [comment]
Should be fixed by #20292.
